src/main/java/com/projects/oop/interfaces/Vehicle.java

```diff
@@ -0,0 +1,13 @@
+package com.projects.oop.interfaces;
+
+interface Vehicle {
+    //all variables and methods are public even if they aren't specified as being public
+    String licensePlate = "Final License Plate";
+    int maxSpeed = 323;
+    default void start(){
+        System.out.println("Starting the vehicle!");
```

Comment:
this would be better with it returned the string and then the caller can decide what to dp